superset/migrations/versions/2023-12-27_13-14_59a1450b3c10_drop_filter_sets_table.py

```diff
@@ -14,17 +14,30 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-USER_OWNER_TYPE = "User"
-DASHBOARD_OWNER_TYPE = "Dashboard"
-
-NAME_FIELD = "name"
-DESCRIPTION_FIELD = "description"
-JSON_METADATA_FIELD = "json_metadata"
-OWNER_ID_FIELD = "owner_id"
-OWNER_TYPE_FIELD = "owner_type"
-DASHBOARD_ID_FIELD = "dashboard_id"
-OWNER_OBJECT_FIELD = "owner_object"
-DASHBOARD_FIELD = "dashboard"
-PARAMS_PROPERTY = "params"
-
-FILTER_SET_API_PERMISSIONS_NAME = "FilterSets"
+"""drop_filter_sets_table
+
+Revision ID: 59a1450b3c10
+Revises: 06dd9ff00fe8
+Create Date: 2023-12-27 13:14:27.268232
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "59a1450b3c10"
+down_revision = "06dd9ff00fe8"
+
+from importlib import import_module
+
+from alembic import op
+
+module = import_module(
+    "superset.migrations.versions.2021-03-29_11-15_3ebe0993c770_filterset_table"
+)
+
+
+def upgrade():
+    module.downgrade()
+
+
+def downgrade():
+    module.upgrade()
```

Comment:
Nice, it never occurred to me we could do something like this 👍

Reply:
I ❤️ DRY code.